builder/cc1as.h

```diff
@@ -89,6 +92,10 @@ struct AssemblerInvocation {
   /// Whether to emit DWARF unwind info.
   EmitDwarfUnwindType EmitDwarfUnwind;
 
+  // Whether to emit compact-unwind for non-canonical entries.
+  // Note: maybe overriden by other constraints.
```

Comment:
```suggestion
  // Note: maybe overridden by other constraints.
```

Reply:
This is code copied from the LLVM project, I'd prefer to keep it as-is (or fix the typo upstream).